aws/resource_aws_ec2_client_vpn_endpoint.go

```diff
@@ -235,6 +237,11 @@ func resourceAwsEc2ClientVpnEndpointRead(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	err = d.Set("tags", tagsToMap(result.ClientVpnEndpoints[0].Tags))
+	if err != nil {
+		return err
```

Comment:
We should return error context here for operators and code maintainers, e.g.

```suggestion
		return fmt.Errorf("error setting tags: %s", err)
```

Reply:
I'll also add error context to the previous 2 calls to `d.Set()` for `authentication_options` and `connection_log_options`.